modules/machinery/physical.py

```diff
@@ -204,13 +214,23 @@ def fog_init(self):
                 "to login into FOG, please configure the correct credentials."
             )
 
+        # Pull out the FOG version from the header and raise error if it is not 1.3.4
+        v = b.find("div",{"id": "version"})
+        runningVer = re.match(r"Running Version\s+(([0-9]+\.)+[0-9]+)",v.text).group(1)
+        if runningVer != "1.3.4":   # This may be better suited to go in lib.cuckoo.common.constants
```

Comment:
You can remove the comment, I think it's just fine here. We're hardcoding the version number anyway.